Client/Cards/list.spec.ts

```diff
@@ -1,8 +1,13 @@
 import * as dotenv from "dotenv"
 import * as pax2pay from "../../index"
+import { ErrorResponse } from "../../model"
+import * as model from "../../model"
+import { Card } from "../Card"
 dotenv.config()
 jest.setTimeout(100000)
 
+const PROVIDERS: model.ProviderCode[] = ["conferma", "ixaris", "wex", "fake", "lodged", "modulr", "unknown", "pax2pay"]
```

Comment:
lodged and unknown aren't proper provider codes (any more), you could delete those